File: codechef/medium/CTMNMF10/solution.cpp
import java.util.Scanner;  // Import Scanner for user input  

class Codechef {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in); // Create Scanner object  
        
        // Prompt the user to enter the number for which the table is generated  
        // Read the user's input directly and use it in the while loop
        int number = scanner.nextInt();  // Read user input using the scanner object

        int multiplier = 1;  // Initialize counter  

        // Loop will run until multiplier becomes equal to 10
        while (multiplier <= 10) {  
            System.out.println(number + " x " + multiplier + " = " + (number * multiplier));  
            multiplier++;  // Increment counter  
        }
        
        scanner.close();  // Close the scanner  
    }
}

File: codechef/medium/CTMNMF10/solution.java
import java.util.Scanner;  

class Codechef {  
    public static void main(String[] args) {
        
        // Create a Scanner object to read input from the user
        Scanner scanner = new Scanner(System.in);  
        
        // Read an integer input from the user
        int number = scanner.nextInt();  
        
        // Variable to store the sum of digits of the number  
        
        int sum=0;


        // Start a while loop, the loop will run until the number becomes 0
        while(number>0){
            
            sum+= number % 10;
            
            number/= 10;
            
        

        }  

        // Print the result, which is the sum of the digits
        System.out.println("Sum of digits: " + sum);
    }  
}